src/mtcp/mtcp_restart.c

```diff
@@ -1168,8 +1168,8 @@ remapExistingAreasToReservedArea(RestoreInfo *rinfo,
 
   size_t num_regions = 0;
 
```

Comment:
The lines below are being changed for all CPUs.

In fact, there had earlier been a bug here, but the latest 'main' branch fixes this bug.  See the commit:  e4b4a82ed51

The earlier bug was quite subtle.  I first saw it on ARM64 (aarch64).  You might be amused by the issue.

So, I believe that we can delete this commit.